sw/device/silicon_creator/rom/e2e/keymgr/rom_e2e_keymgr_init_test.c

```diff
@@ -22,6 +23,8 @@
 
 OTTF_DEFINE_TEST_CONFIG();
 
+static uint32_t otp_state[1024] = {0};
```

Comment:
Is there a named constant that could be used instead of 1024? Or at least check in the code that `kOtpPartitions[p].size` is smaller than the size of this array.

Reply:
Good ideas. Fixed.